content/docs/studio/get-started.md

```diff
@@ -100,8 +100,9 @@ You should now see that a project has been added in your dashboard.
 
 ## Explore and share your projects
 
-1. After creating a project, you can use it to [explore the ML experiments] as
-   well as [submit new experiments] from Iterative Studio.
+1. After creating a project, you can use it to [explore the ML experiments],
+   [set up your model training CI job], and [submit new experiments] from
+   Iterative Studio.
```

Comment:
In general, should we mention the CML name/brand more? So far the docs use descriptive phrasing instead ("training CI job" in this case).

Reply:
Changed to `[set up your model training CI job, potentially using CML]`. Because users have the choice not to use CML, we can't really say that they should create CI actions with CML.

Reply:
OK